openid-sharedsignals-framework-1_0.md

```diff
@@ -240,10 +266,20 @@ Subject Principals are identified by Subject Members defined below.
 
 # Subject Members in SSF Events {#subject-ids}
 
-A member of type Subject in an SSF event MAY have any claim name. Each Subject Member MUST
-refer to exactly one Subject Principal.
+## Subject Members {#subject-members}
+A Subject Member of a SSF event describes a subject of the event. A top-level claim named `sub_id` SHOULD be used to describe the primary Subject Principal of the event.
```

Comment:
Can we say MUST instead of SHOULD? We can make the `sub_id` claim required without messing with the CAEP or RISC `subject` claim.

Reply:
I think the concern was that if there are existing Transmitters that don't insert the sub_id claim, then they will become incompatible with the updated spec.

Reply:
The downside of this decision is that Receivers will forever have to look in two places to find the subject information and, because it is not required in either place, it would be possible (although unlikely) for a Transmitter to send a SET with no subject information.

This draft already breaks backwards compatibility. I think our goal should be no breakage _without a good reason_.

Reply:
We can define an interoperability profile which can be more definitive about whether or not this field MUST exist.

Reply:
I agree that SHOULD needs to be MUST for `sub_id`

Reply:
updated to "MUST"